core/arch/arm/plat-stm32mp1/pm/psci.c

```diff
@@ -167,6 +176,31 @@ int psci_cpu_on(uint32_t core_id, uint32_t entry, uint32_t context_id)
 
 	return rc;
 }
+
+/* Override default psci_cpu_off() with platform specific sequence */
+int psci_cpu_off(void)
+{
+	unsigned int pos = get_core_pos();
+	uint32_t excep;
+
+	if (pos == 0) {
+		EMSG("PSCI_CPU_OFF not supported for core0, use system_off");
```

Comment:
`s/core0/core #0/`
`s/system_off/PSCI_CPU_OFF/`

Just curious, is this true even if other CPUs are still running?

Reply:
Actually the chosen scheme for power sequences expect boot core (core #0) is the one that boots up first and is powered off last (system_off, system_suspend). When secondary boot core shall be deactivated, we expect a call to `psci_cpu_off()` and the core will be reset until being woken back, which cannot be done for core #0.